src/index.html

```diff
@@ -7,6 +7,423 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="MOYO Logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops&computers"
+              data-qa="nav-hover"
+            >
+              Laptops&nbsp;&&nbsp;Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card" data-qa="card">
+        <div class="card__img-wrapper">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+            class="card__img"
+          >
+        </div>
+
+        <section class="card__info">
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
```

Comment:
```suggestion
              <div class="stars__star"></div>
              <div class="stars__star"></div>
```

Add blank line only between multiple sibling blocks of html and not between single tags

example:

```
<li>
  <a>first link</a>
</li>

<li>
  <a>second link</a>
</li>
```